### ISSUE TITLE
Farming Milkweed should leave a hardened row

### ISSUE THREAD

**Issue by Arcurus** [jasonrohrer/OneLifeData7#1176]
Making ropes with farmed milkweed is already very expensive. Leaving hardened rows for domestic planted milkweed would at least save half of the soil.

**Comment by DougLefelhocz** [comment]
> Yeah, this is intentional, because milkweed is a wild-ish plant, and there is no domesticated version.

[Link](https://github.com/jasonrohrer/OneLifeData7/issues/1074)

> Well, the idea here is that there is no such thing as "domesticated" milkweed. It is a wild plant, and thus behaves like one after you plant it. In fact, it is exactly the same object. It will go through the same cycles as regular wild milkweed.

> This is a bit like planting trees, most of which do not have domesticated versions.

> It is a tough trade-off with early clothing (which would actually cause you to consume less food if you had it).

> But it has been this way in the game forever. Milkweed is just one of those precious things.... just like in real life.

[Other link](https://github.com/jasonrohrer/OneLifeData7/issues/1145)

**Comment by Shadie116** [comment]
Yeah, and can it also be in pairs of two considering how time consuming it is to take one at a time for a single fraction of string making material

**Comment by Butterbraugh** [comment]
The real solution would be to add a domestic alternative to milkweed. How about a plant that was actually cultivated specifically for use in making rope throughout human history. One that is factually cheap and easy to grow in large amounts and provides massive amount of plant fibers when harvested. Hemp.

**Comment by Shadie116** [comment]
Honestly good idea

> The real solution would be to add a domestic alternative to milkweed. How about a plant that was actually cultivated specifically for use in making rope throughout human history. One that is factually cheap and easy to grow in large amounts and provides massive amount of plant fibers when harvested. Hemp.
> 
> You could have the wild plant not be useful except the seeds like many domestic plants in game. This method could allow for hardened rows. Or you could require steel tools (knife) in order to harvest from the wild to make it distinct from milkweed for eve camps.
> 
> You could receive one hemp stalk per plant grown which would need to be pounded on a flat rock with a stone to make a pile of hemp fibers. (4-6 per plant) that could be combined into strings and ropes.

**Comment by Rukhmar** [comment]
> The real solution would be to add a domestic alternative to milkweed. How about a plant that was actually cultivated specifically for use in making rope throughout human history. One that is factually cheap and easy to grow in large amounts and provides massive amount of plant fibers when harvested. Hemp.
> 
> You could have the wild plant not be useful except the seeds like many domestic plants in game. This method could allow for hardened rows. Or you could require steel tools (knife) in order to harvest from the wild to make it distinct from milkweed for eve camps.
> 
> You could receive one hemp stalk per plant grown which would need to be pounded on a flat rock with a stone to make a pile of hemp fibers. (4-6 per plant) that could be combined into strings and ropes.

This. But if you don't want to get into the cannabis side of it, you can use another plant that has been used traditionally for fibers as well: Flax. Milkweed as it is, is a huge waste of resources in almost all cases since there is no benefits to planting it over the wild version, just huge costs.

**Comment by Shadie116** [comment]
AHHAGUAH Cannabis weed
